src/components/nodes/pool/pool.vue

```diff
@@ -177,35 +201,40 @@ export default {
 
       lanes.push(this.pushNewLane());
 
-      await Promise.all(lanes);
+      await Promise.all(lanes).
+        then((val) => {
+          if (emitMultiplayer && this.$parent.isMultiplayer) {
+            window.ProcessMaker.EventBus.$emit('multiplayer-addLanes', val);
+          }
+        });
       this.$emit('set-shape-stacking', this.shape);
       this.graph.getLinks().forEach(link => {
         this.$emit('set-shape-stacking', link);
       });
       this.$emit('save-state');
     },
-    createLaneSet() {
+    createLaneSet(id) {
       const laneSet = this.moddle.create('bpmn:LaneSet');
       this.laneSet = laneSet;
       const generator = this.nodeIdGenerator;
-      const [laneSetId] = generator.generate();
+      const [laneSetId] =  id ? id : generator.generate();
```

Comment:
I believe this can be done 
```suggestion
      const [laneSetId] =  id ?? generator.generate();
```